editions/tw5.com/tiddlers/community/Contributor License Agreement.tid

```diff
@@ -1,10 +1,10 @@
 created: 20150630205511173
-modified: 20220226175543038
+modified: 20230803053548871
 tags: 
 title: Contributor License Agreement
 type: text/vnd.tiddlywiki
 
-Like other OpenSource projects, TiddlyWiki5 needs a signed contributor license agreement from individual contributors. This is a legal agreement that allows contributors to assert that they own the copyright of their contribution, and that they agree to license it to the UnaMesa Association (the legal entity that owns TiddlyWiki on behalf of the community).
+Like other OpenSource projects, TiddlyWiki5 needs a signed contributor license agreement from individual contributors. This is a legal agreement that allows contributors to assert that they own the copyright of their contribution, and that they agree to license it to the [[UnaMesa|https://unamesa.org/]] Association (the legal entity that owns TiddlyWiki on behalf of the community).
```

Comment:
If we have identical external links imo it's better to create a new tiddler: `UnaMesa` tagged: `Definitions` which contain a brief info about the organisation which then links to the external links. IMO we want to keep users on the page instead of sending them to external sites. 

This may also be true for the "History of TiddlyWiki" tiddler and the external links.

Reply:
Added an UnaMesa tid with some summary info from the site, tweaked this and TiddlyHistory